pkg/cloudevents/transport/amqp/codec_test.go

```diff
@@ -508,13 +501,6 @@ func TestCodecAsMiddleware(t *testing.T) {
 					got.Data = data
 				}
 
-				if tc.wantErr != nil || err != nil {
```

Comment:
the tests need to be expanded to non-happy path only. Please leave this?

Reply:
will do, although right now it is unreachable.